Rollbar/RollbarConfiguration.h

```diff
@@ -57,61 +51,58 @@ typedef NS_ENUM(NSUInteger, CaptureIpType) {
 
 - (NSDictionary *)customData;
 
-@property (readwrite, atomic) BOOL enabled;
-@property (atomic, copy) NSString *accessToken;
-@property (atomic, copy) NSString *environment;
-@property (atomic, copy) NSString *endpoint;
+@property (nonatomic) BOOL enabled;
```

Comment:
I assume moving from atomic to nonatomic is for performance reason in the following multiple instances?

Reply:
I am still up in the air on how I want to handle these. The use of atomic is not useful if the class is not actually thread safe. This object isn't really thread safe in the sense of causally serializing reads and writes, so why pay the overhead of the lock on each property. If we do want them to be atomic then we should leave it off as it is the default. The Objective C style is to only add the modifiers that are necessary.